react/react1/week3/todo-app/src/components/TodoItem/TodoItem.jsx

```diff
@@ -0,0 +1,85 @@
+import { useState } from "react";
+import PropTypes from "prop-types";
+import WrapperTodo from "../WrapperTodo/WrapperTodo";
+
+const TodoItem = ({ todo, handlerDelete, updateTodo }) => {
+  const { id, description, deadline } = todo;
+  //   console.log(typeof id);
+  //   const normalizeIdType = typeof id === "string" ? id : toString(id);
+  //   console.log(typeof normalizeIdType);
+
+  const [isCheck, setIsCheck] = useState(false);
```

Comment:
I'd rather that we have the checked state on the todo object, so that it'll keep its state no matter the state of the TodoItem component.